anti-work/shortest
# Feat: Refine shortest API

v0.0.4 

Changes: 
- Simplify shortest API
- Improve Test Reporting 
- Improve Navigation 
- Simplify shortest.config.ts file
- Remove browser session persistence 
- Add support for playwrights page object 
- Move screenshot folder to .shortest and add periodic cleanup
- Update docs

dashboard.test.ts
```typescript
import { test } from '@antiwork/shortest';

const githubCredentials = {
    username: process.env.GITHUB_USERNAME,
    password: process.env.GITHUB_PASSWORD,
}
  
test('Login to the app using Github login', githubCredentials)
test('clicking write new test button should initiate test generation')
```

shortest.config.ts
```typescript
import type { ShortestConfig } from '@antiwork/shortest';

export default {
  headless: false,
  baseUrl: 'http://localhost:3000',
  testDir: 'app/__tests__',
  anthropicKey: process.env.ANTHROPIC_API_KEY
} satisfies ShortestConfig; 
```
 

https://github.com/user-attachments/assets/751eb284-b7d8-4ad2-83f6-c9b3779c5f26
